api/controllers/v0.3/endpoints.js

```diff
@@ -446,12 +449,18 @@ async function checkDependentEligibility(req, res) {
     status = 200;
   } catch (error) {
     response = modelResponse.errorResponseData(
-      collectErrorResponseData(error.status || 400, "", error.message, "", "") // eslint-disable-line comma-dangle
+      collectErrorResponseData(
+        error.status || 400,
+        error.type || "",
+        error.message,
+        "",
```

Comment:
What are the empty strings for?

Reply:
This is from the `collectErrorResponseData` function and the last two parameters are for `title` and `debugMessage`. For this specific example, the error object will be something like:
```
{
    "status": 400,
    "type": "unavailable-username",
    "message": "This username is unavailable. Please try another.",
}
```

and `title` and `debugMessage` aren't necessary. It also adds a `detail` key which isn't used. The most important things are returned right now and I'll remove the rest in another pass. It was already in the code but I don't want to delete it now in case it breaks any previous endpoints or existing functions.